src/OrchardCore/OrchardCore.Module.Targets/OrchardCore.Module.Targets.props

```diff
@@ -16,9 +16,20 @@
     <EnableDefaultNoneItems>false</EnableDefaultNoneItems>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <ResolvedRazorCompileToolset>RazorSdk</ResolvedRazorCompileToolset>
+    <EnableDefaultRazorGenerateItems>false</EnableDefaultRazorGenerateItems>
+    <RazorCompileOnBuild>true</RazorCompileOnBuild>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="$(MicrosoftAspNetCoreMvcPackageVersion)" />
```

Comment:
Why? Modules would always mvc.

Reply:
It will be removed, right now it is required here to hit the `RazorSdk` when building a module.

But it will be removed when we will be able to use directly `<Project Sdk="...RazorSdk">`